test/integration/editor/new.spec.ts

```diff
@@ -44,6 +44,10 @@ test('can create new component', async ({ page }) => {
 
   await editorModel.createPage('somePage');
   await editorModel.createComponent('someComponent');
+  await editorModel.componentCatalog.hover();
+  await expect(editorModel.componentCatalog.getByText('someComponent')).toBeVisible();
+  await editorModel.dragNewComponentTo('someComponent');
```

Comment:
I'll rechange the name of this method to `dragNewComponentToCanvas`, it makes it more readable (I had changed the name some day ago).